index.html

```diff
@@ -70,16 +68,113 @@ <h5>Skills</h5>
             </ul>
           </div>
         </div>
-
+        <hr />
         <div class="links">
           <a><i class="fa fa-github"></i></a>
           <a><i class="fa fa-twitter"></i></a>
           <a><i class="fa fa-envelope"></i></a>
           <a><i class="fa fa-linkedin"></i></a>
         </div>
+      </section>
+
+      <section class="works projects">
+        <span class="works1"></span>
+        <span class="works2"></span>
+        <div class="down">
+          <i class="fa fa-light fa-arrow-down"></i><span>Work</span>
+        </div>
+        <h3>MY RECENT</h3>
+        <h3>WORK</h3>
```

Comment:
- [ ] Hi, this `.works.projects` section should use `display: grid` as per the requirements. please use `grid` in order to become accustomed to how it works.

Reply:
👍🏽 This is well implemented.